lib/main.js

```diff
@@ -171,14 +171,28 @@ export default {
   },
 
   activate: () => {
+    var help_data = '';
     require("atom-package-deps").install('linter-puppet-lint');
 
     // Check if puppet-lint has support for the %{column} placeholder
     var command = atom.config.get("linter-puppet-lint.executablePath");
     var version = require('child_process').spawn(command,['--help']);
```

Comment:
Why not just replace this mess with a call to `helpers.exec()`? (That would need to be moved to a global scope)